src/validator-guides/how-to-setup-a-validator-node.md

```diff
@@ -0,0 +1,79 @@
+# How to setup a validator node
+
+For this instruction we consider a cloud server running Linux. The validator can use any provider according to his preferences . As OS it is best to use a recent Debian Linux. For this guide we will be using Ubuntu 22.04, but the instructions should be similar for other platforms.
+
+Please make sure to follow all of the instructions! Mistakes in setting up a node may get your tokens slashed and may put the tokens of your nominators at risk. Check our [Validator programme](sydney-validator-programme.md) for requirements and incentives.
+
+#### Software requirements
+
+The following dependencies are required to run the project:
+
+* rust, wasm32-unknown-unknown target
+* protobuf
+* dylint
+
+Below is the code example how to install required software:
+
+```
+# Install rust
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh 
+
+# Install wasm32-unknown-unknown target
+rustup target add wasm32-unknown-unknown
+rustup component add rust-src
+
+# Install protobuf
+sudo apt install protobuf-compiler
+
+# Install dylint
+cargo install cargo-dylint dylint-link
+```
+
+You can use docker container to setup a validator node. Due to the highly CPU dependent nature of 'cargo build' command, it's strongly recommended that you have at least 8 core enabled for this method. It takes around 20 mins to complete with this suggested requirements, exponentially more if you use lesser proccessor power during the docker build operation.
+
+From the repository's root directory execute following commands in order:
```

Comment:
We need to include somewhere that the user must `git clone https://github.com/GoldenGateGGX/golden-gate.git`

Reply:
We also need to install docker...?

Reply:
Make sure in the documentation that we ensure the user is part of the `docker` group.  To add the user to the docker group run the command `sudo usermod -aG docker $USER`